automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/user/RegistrationSpec.scala

```diff
@@ -59,36 +79,13 @@ class RegistrationSpec extends FreeSpec with BeforeAndAfter with Matchers with W
         state = "MA",
         country = "USA")
 
-      new DataLibraryPage().validateLocation()
-    }
-  }
+      await ready new DataLibraryPage()
```

Comment:
it this correct that it is waiting for the data library page? or should it be workspace list page?

Reply:
It is correct. For a new user when they first register, they are taken to the data library page. On subsequent sign ins they are taken to the workspace list page. Isaac implemented this based on a PO request a number of months ago so ask him if you're curious and want to know more.